cranelift/filetests/filetests/runtests/selectif-spectre-guard.clif

```diff
@@ -314,3 +314,13 @@ block0(v0: i8, v1: i128, v2: i128):
 ; run: %select_spectre_guard_i128_sle(127, 32, -1) == -1
 ; run: %select_spectre_guard_i128_sle(127, 32, 19000000000000000000) == 19000000000000000000
 ; run: %select_spectre_guard_i128_sle(42, 32, 19000000000000000000) == 32
+
+function %select_spectre_guard_i128_cond(i128, i128, i128) -> i128 {
```

Comment:
This file is configured to run on aarch64 as well as x64, so I'm expecting CI to fail when it tries to run this test on aarch64. I assume we can't add this test here without fixing at least #5453 as well.

Reply:
Hmm, yeah, I'll take a look at that...

Reply:
Updated!